src/Scene/NodeProcess.js

```diff
@@ -344,31 +333,29 @@ function updateNodeElevation(quadtree, node, layersConfig, force) {
     //   * we already have 1 texture (so currentElevation >= 0)
     // Again, LayeredMaterial's 1 elevation texture limitation forces us to `break` as soon
     // as one layer can supply a texture for this node. So ordering of elevation layers is important.
-    if (bestLayer == null) {
-        for (let i = 0; i < elevationLayers.length; i++) {
-            let layer = elevationLayers[i];
-
-            if (!layer.tileInsideLimit(node, layer)) {
-                continue;
-            }
+    for (let i = elevationLayers.length - 1; i >= 0; i--) {
```

Comment:
I don't really mind changing the order here, but if it's important add a comment stating why we must browse elevation layer in this order.